### PR TITLE
feat(sass functions): fix --var function

### LINKED ISSUE #5572: [Sass] "--var" function doesn't work when $legacy-support is set to false on production build.
## Description  
`--var` function doesn't work when `$legacy-support` is set to false on production build.

 * igniteui-angular version: 8.1
 * browser: Modern browsers

## Steps to reproduce  

1. Unzip the attached sample.
2. npm ci
3. npm run build:prod
4. npx http-server .\dist\my-app
5. Browse  http://10.30.0.133:8080

## Result  
Background color is not applied to the navbar. I found that `var()` has commna but doesn't have second parameter, which is invalid syntax.
image

It seems that [this commit](https://github.com/IgniteUI/igniteui-angular/commit/9a59ffbff611c7fcd24358eb404eeded4e8f5b45) causes this issue.

## Expected result  
"--var" function should be compiled to valid var syntax.

## Attachments  
[my-app.zip](https://github.com/IgniteUI/igniteui-angular/files/3460481/my-app.zip)

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #5572 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss [hunk 1]
```diff
@@ -354,7 +354,11 @@
 
     @if map-has-key($map, $key) {
         @if $igx-legacy-support == false {
-            @return unquote('var(--#{map-get($map, 'name')}-#{$key}, #{$fallback})');
+            @if ($fallback != '') {
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss]:
```suggestion
            @if not($fallback) {
```

File: projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss [hunk 2]
```diff
@@ -354,7 +354,11 @@
...
+                @return unquote('var(--#{map-get($map, 'name')}-#{$key}, #{$fallback})')
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss]:
```suggestion
                @return unquote('var(--#{map-get($map, 'name')}-#{$key})');
```

File: projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss [hunk 3]
```diff
@@ -354,7 +354,11 @@
...
+            } @else {
+                @return unquote('var(--#{map-get($map, 'name')}-#{$key})')
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/base/utilities/_functions.scss]:
```suggestion
               @return unquote('var(--#{map-get($map, 'name')}-#{$key}, #{$fallback})');
```

**Comment by zdrawku** [comment]
@tkiryu if you've tested the PR, could you set the according label to it, in order to proceed with a merge. The flow is awaiting-test -> in-test -> verified

**Comment by tkiryu** [comment]
@zdrawku , I tested the PR. I proceeded the status to `verified`. Is that OK?

**Comment by zdrawku** [comment]
Yes, thank you :)

**Pull request merged**